### PR TITLE
Add 'Origami Studio' App

### PR DESCRIPTION
**Pull request by olierxleben**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [ ] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

added the latest and officially first release of Origami Studio to the homebrew cask list

### CODE REVIEW

**Comment by miccal** [comment]
You need to adda [comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) above the url stanza.

File: Casks/origami-studio.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'origami-studio' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://scontent-sea1-1.xx.fbcdn.net/t39.2270-6/10000000_930050417126110_4686042515185336320_n.dmg/Origami-Studio-External-Mac_42534415.dmg'
```
Comment (vitorgalvao) [inline Casks/origami-studio.rb]: `42534415` looks like a version.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/origami-studio.rb (hunk 1 above)
Comment (adidalal) [inline Casks/origami-studio.rb]: In this case, I actually recommend the un versioned url - https://fb.me/getorigamistudio
It's a facebook cdn url, those break after some time, I believe.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ec8e47baf775fe383fa64df5e2e3bae24312cf9c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**